workspaces/arborist/test/query-selector-all.js

```diff
@@ -712,6 +712,7 @@ t.test('query-selector-all', async t => {
     [':semver(^1.1.1, [version], lte)', []],
     [':semver(^14.0.0, :attr(engines, [node]), intersects)', ['bar@2.0.0']],
     [':semver(>=14, :attr(engines, [node]), subset)', ['abbrev@1.1.1', 'bar@2.0.0']],
+    [':semver(>=14, :attr(engines, [node]), rsubset)', ['bar@2.0.0']],
```

Comment:
naive question but is this functionally different than `':semver(:attr(engines, [node]), >=14, subset)'`